tensorflow/python/kernel_tests/mkl_ops_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+  Tests to verify if the conversion of python ops to MKL graph ops happen when
+  MKL is enabled. These tests create graphs to verify the rewrite, and
+  therefore will not test eager rewrites. The tests will be skipped if MKL is
+  not enabled.
+"""
+
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import test_util
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.platform import test
+from tensorflow.core.protobuf import config_pb2
+from tensorflow.core.protobuf import rewriter_config_pb2
+
+config = config_pb2.ConfigProto()
+run_options = config_pb2.RunOptions(
+    trace_level=config_pb2.RunOptions.FULL_TRACE, output_partition_graphs=True)
+config.graph_options.rewrite_options.constant_folding = (
+    rewriter_config_pb2.RewriterConfig.OFF)
+config.graph_options.optimizer_options.opt_level = -1
+
```

Comment:
Please add another blank line. There should be two blank lines between top-level definitions [[style guide](http://google.github.io/styleguide/pyguide.html#35-blank-lines)]. Same for other places.